### PR TITLE
Geth support

### PR DESCRIPTION
**Pull request by kajoseph**
This PR depends on https://github.com/bitpay/bitcore/pull/3388

### CODE REVIEW

File: packages/bitcore-node/test/integration/ethereum/csp.spec.ts [hunk 1]
```diff
@@ -333,66 +333,11 @@ describe('Ethereum API', function() {
       await streamWalletTransactionsTest(chain, network, true)
     );
 
-    it('should stream DEX wallet transactions', async () => {
-      await EthBlockStorage.collection.insertMany(EthBlocks as any);
-      await EthTransactionStorage.collection.insertMany(EthTransactions as any);
+    it('should stream DEX wallet transactions with parity trace blocks', async () => {
```
Comment (escottalexander) [inline packages/bitcore-node/test/integration/ethereum/csp.spec.ts:336]: Should this be 'erigon trace blocks'?

**Review by escottalexander — APPROVED** [review]
Looks good. Just a few small things I noticed.

File: .docker/geth-keystore/UTC--2022-06-30T13-15-12.406902396Z--ec12cd1ab86f83c1b26c5caa38126bc4299b6cba [hunk 2]
```diff
@@ -0,0 +1 @@
+{"address":"ec12cd1ab86f83c1b26c5caa38126bc4299b6cba","crypto":{"cipher":"aes-128-ctr","ciphertext":"cacd04d2a934c14eb94f0b6a56b22909c3b35fca2503a3e821df9ccd3606016d","cipherparams":{"iv":"ff1e71097f717fa3f11d388253a6a227"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":4096,"p":6,"r":8,"salt":"f50417f15ba7921b08cc93cce68dfe2b1062fe73162f543a1c4876ac0dc46ffc"},"mac":"815724427e62715487fb091251074cb4b2837f6bc94273f7d3cd194b22bbb6e7"},"id":"1aa36c20-312f-41ba-9462-914b8020201b","version":3}
```
[the remark this answers is not in the document]
Comment (kajoseph) [reply .docker/geth-keystore/UTC--2022-06-30T13-15-12.406902396Z--ec12cd1ab86f83c1b26c5caa38126bc4299b6cba:1]: No, this is for tests.

**Review by escottalexander — APPROVED** [review]
This set of changes looks good.

**Review by nitsujlangston — APPROVED** [review]

**Pull request merged**